src/main/java/frc/robot/commands/ExampleCommand.java

```diff
@@ -9,16 +9,12 @@
 
 /** An example command that uses an example subsystem. */
 public class ExampleCommand extends CommandBase {
-  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
```

Comment:
Why are you editing ExampleCommand.java?  If you want to remove the file (since it's only an example), you should probably do that in a separate pull request.  But there's no reason to remove the @SuppressWarnings directive and the ExampleSubsystem.